src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/ExternalMovement.java

```diff
@@ -12,7 +12,6 @@
 import org.hibernate.type.YesNoConverter;
 import jakarta.persistence.Convert;
 
-import javax.annotation.Nullable;
```

Comment:
this is an odd one.  `@Nullable` is a suggestion and isn't enforced.  However it is used by the kotlin null checker and an exception is thrown if a field is not marked as `Nullable`, so the tests then fail.  Reading https://youtrack.jetbrains.com/issue/KT-54205/Support-jakarta-Nullability-annotations aren't intending to support the jakarta annotation, but do currently support the jetbrains one so I've switched to that one instead.  Looks like https://jspecify.dev/ hasn't got off the ground yet so can't switch to that one either.